samples/tracing/src/main/java/com/example/tracing/api/TracingEndpoint.java

```diff
@@ -1,26 +1,38 @@
 package com.example.tracing.api;
 
 import akka.Done;
+import akka.http.javadsl.model.HttpResponse;
+import akka.javasdk.Tracing;
+import akka.javasdk.annotations.Acl;
 import akka.javasdk.annotations.http.HttpEndpoint;
 import akka.javasdk.annotations.http.Post;
 import akka.javasdk.client.ComponentClient;
-import com.example.tracing.application.TracingAction;
+import akka.javasdk.http.HttpResponses;
 import akka.javasdk.timer.TimerScheduler;
+import com.example.tracing.application.TracingAction;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.StatusCode;
 
 import java.time.Duration;
+import java.util.Optional;
 import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
 
+// Opened up for access from the public internet to make the sample service easy to try out.
+// For actual services meant for production this must be carefully considered, and often set more limited
+@Acl(allow = @Acl.Matcher(principal = Acl.Principal.INTERNET))
 @HttpEndpoint("/tracing")
 public class TracingEndpoint {
 
     private final ComponentClient componentClient;
     private final TimerScheduler timerScheduler;
+    private final Tracing spanFactory;
```

Comment:
why calling it `spanFactory` and not simply `tracing`?

Reply:
Leftover, I'll fix